### DISCUSSION TITLE
How to get the Server URL

### DISCUSSION THREAD

**Discussion by oyebayofemi** [saran-surya/email_auth_node#30] [Q&A]
I am getting this error:-

--- Package Error ---
I/flutter (20207): Unable to access remote server. 😑
I/flutter (20207): --- End Package Error ---
I/flutter (20207): --- package Error ---
I/flutter (20207): email-auth >> The remote server is not a valid.
I/flutter (20207): email-auth >> configured server : "[https://testingemailauthverification.herokuapp.com/]()"
I/flutter (20207): --- package Error ---

I also tried the link 'https://mail-abot.herokuapp.com/ ' same error

How do i get the server link??

**Comment by saran-surya** [comment]
Hi @oyebayofemi ,
https://testingemailauthverification.herokuapp.com This is your server URL.
Please read the docs. Thankyou
